### DISCUSSION TITLE
GPU acceleration

### DISCUSSION THREAD

**Discussion by miaodd98** [OpenStitching/stitching#90] [Ideas]
Hi! I've find this repo useful for image stitching. Is there any chance for this code to be run on a GPU for acceleration? I've found the 'try_use_gpu' attributes and changed it into True but nothing happened. Could you please help this out?

**Comment by lukasalexanderweber** [comment]
I'm not experienced in this, if you find out any stuff I would be glad if you let me know

**Comment by DiTo97** [comment]
> Hi! I've find this repo useful for image stitching. Is there any chance for this code to be run on a GPU for acceleration? I've found the 'try_use_gpu' attributes and changed it into True but nothing happened. Could you please help this out?

GPU acceleration is available only for the C++ stitching module, as OpenCV's Python bindings do not yet support acceleration.

I found a roadmap for the Python support on the OpenCV website some time ago. I will share the link if I find it again.
